crates/derive/src/online/pipeline.rs

```diff
@@ -18,7 +18,7 @@ pub type OnlinePipeline =
 /// An `online` Ethereum data source.
 pub type OnlineDataProvider = EthereumDataSource<
     AlloyChainProvider,
-    OnlineBlobProvider<OnlineBeaconClient, SimpleSlotDerivation>,
+    OnlineBlobProviderWithFallback<OnlineBeaconClient, OnlineBeaconClient, SimpleSlotDerivation>,
```

Comment:
I don't really mind doing this, but can we make the fallback optional so that the current functionality is backwards compatible?